apiary.apib

```diff
@@ -399,6 +399,7 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### Latest
 
+- We added `is_duration_time_tracking` on `timeTracking.list`
```

Comment:
I think the stuff below here probably needs a `### July` header

Reply:
Consider it done.